xml/decoders/Zimo_Unified_software_v32_MX695_696_697V.xml

```diff
@@ -154,6 +156,7 @@
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV1-CV99.xml"/>
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV100-CV152version30.xml"/>
       <xi:include href="http://jmri.org/xml/decoders/zimo/CV152bit7version27.xml"/>
+      <xi:include href="http://jmri.org/xml/decoders/zimo/CV152-CV157version28.xml"/>      
```

Comment:
I think that this should be `CV153-CV157version28.xml` as this is failing validation.